mlir/lib/Target/KokkosCpp/TranslateToKokkosCpp.cpp

```diff
@@ -28,6 +28,7 @@
 #include <utility>
 
 #include <iostream>
+//#include <unistd.h>
```

Comment:
This was used by the "pause to attach debugger" code, see below. Will enable permanently in a future PR